src/sentry/models/release.py

```diff
@@ -157,6 +158,7 @@ class Meta:
         index_together = (
             ("project", "adopted"),
             ("project", "unadopted"),
+            ("project", "first_seen_transaction"),
```

Comment:
Are we actually going to search on this col?

Reply:
Yes I am going to need to query that and add the delay to it to determine if we are currently after that deadline, and if so don't send this sample rate boost to relay (This should happen on ProjectConfig invalidation)